src/app/services/auth.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@angular/core';
+
+import { UserManager, UserManagerSettings, User } from 'oidc-client';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+
+  private manager = new UserManager(getClientSettings());
+  private user: User = null;
+  constructor() {
+    this.manager.getUser().then(user => {
+      this.user = user;
```

Comment:
we should use RxJs for stroring and maipulating global values (user is global thing).